integration/v3_grpc_inflight_test.go

```diff
@@ -54,6 +55,101 @@ func TestV3MaintenanceDefragmentInflightRange(t *testing.T) {
 	<-donec
 }
 
+// TestV3CompactInflightHashKV ensures that HashKV returns
+// correct hash while compacting.
+func TestV3CompactInflightHashKV(t *testing.T) {
```

Comment:
The test in mvcc would use a kvstore to test HashByRev; the test in integrations/ would test that the server responds to the RPC